.github/workflows/atd_moped_build_stage.yml

```diff
@@ -155,3 +155,36 @@ jobs:
           echo "PWD: $(pwd)"
           source $(pwd)/.github/workflows/aws-moped-api-helper.sh
           deploy_moped_api
+
+  build_moped_cognito_at_edge:
+    name: Build Moped cognito-at-edge
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+          architecture: "x64"
+      # Get the code first
+      - name: "Checkout"
+        uses: actions/checkout@v2
+      # Then install the AWC CLI tools & boto3
+      - name: "Install AWS Cli"
+        run: |
+          sudo apt-get install python3-setuptools
+          pip3 install awscli boto3
+      # Run the shell commands using the AWS environment variables
+      - name: "Build"
+        env:
+          AWS_DEFAULT_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          IGNORE_BRANCH: "TRUE"
+        run: |
+          export BRANCH_NAME=${GITHUB_REF##*/}
+          echo "SHA: ${GITHUB_SHA}"
+          echo "ACTION/BRANCH_NAME: ${BRANCH_NAME}"
+          echo "GR: ${GITHUB_REF}"
+          echo "PWD: $(pwd)"
+          source $(pwd)/.github/workflows/aws-moped-cognito-edge-helper.sh
+          build_lambda_function
+          deploy_lambda_function
```

Comment:
This file creates a job to build the cognit-at-edge in moped-test (but does not deploy).